src/components/single-place/comment-view-section/comment-view-item/comment-view-item.js

```diff
@@ -1,18 +1,71 @@
 import React, { Component } from 'react';
 import './comment-view-item.scss';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faStar } from '@fortawesome/free-solid-svg-icons';
+import { faUserAstronaut } from '@fortawesome/free-solid-svg-icons';
 
 export default class CommentViewItem extends Component {
+  dateConvert = (date) => {
+    let now = new Date()
+    let d = new Date(date);
```

Comment:
d - bad  name. It is unclear what it means.
A variable name should have a clean, obvious meaning, describing the data that it stores.

Reply:
Done